subsys/usb/device/usb_device.c

```diff
@@ -508,6 +513,38 @@ static bool usb_get_descriptor(struct usb_setup_packet *setup,
 	return found;
 }
 
+/*
+ * @brief Get 32-bit endpoint mask from index
+ *
+ * In the returned 32-bit word, the bit positions in the lower 16 bits
+ * indicate OUT endpoints, while the upper 16 bits indicate IN
+ * endpoints
+ *
+ * @param [in]  ep Endpoint of interest
+ *
+ * @return 32-bit bitmask
+ */
+static uint32_t get_ep_mask_from_addr(uint8_t ep)
+{
+	uint32_t ep_mask = 0;
+	uint8_t ep_idx;
+
+	ep_idx = ep & (~USB_EP_DIR_IN);
+	if (ep_idx > 15) {
+		LOG_ERR("Endpoint 0x%02x is invalid", ep);
+		goto done;
+	}
+
+	ep_mask = 1 << ep_idx;
```

Comment:
ep_mask = BIT(ep_idx);

you can remove the empty line below, or actually, I find this variant more readable:

```c
	if (ep & USB_EP_DIR_IN) {
		ep_bm = BIT(ep_idx + 16);
	} else {
		ep_bm = BIT(ep_idx);
	}
```

Reply:
Thanks - changed